src/Core/Checkout/Promotion/Cart/Discount/Filter/Exception/FilterSorterNotFoundException.php

```diff
@@ -6,6 +6,9 @@
 use Shopware\Core\Framework\ShopwareHttpException;
 use Symfony\Component\HttpFoundation\Response;
 
+/**
+ * @deprecated tag:v6.8.0 - reason:remove-exception - Will be removed, use PromotionException::filterSorterNotFound() instead
+ */
 #[Package('checkout')]
 class FilterSorterNotFoundException extends ShopwareHttpException
```

Comment:
For an easier transition, these exceptions should start with the domain exception and inherit the error code